rero_ils/modules/cli.py

```diff
@@ -159,3 +162,23 @@ def schedules():
     for key, value in celery_ext.celery.conf.beat_schedule.items():
         click.echo(key + '\t', nl=False)
         click.echo(value)
+
+
+@utils.command()
+@click.option('--force', is_flag=True, default=False)
+@with_appcontext
+@es_version_check
+def init(force):
+    """Initialize registered templates, aliases and mappings."""
```

Comment:
As it is a invenio patch. You can explain why it is needed and you can also add a TODO for future removal, eventually linked to an issue number on `invenio-search`.

Reply:
done